commands/heroku/heroku-ps.md

```diff
@@ -0,0 +1,25 @@
+### List running dynos
+
+`heroku ps {{TYPE}} --app {{APP_NAME}}`
+
+- <b>TYPE</b>: Type of dyno.  This is optional. If it is not given then all the dynos are listed.
+- <b>APP_NAME</b>: Name of the app.
+
+#### Example:
+
+`heroku ps --app TestApp`
+
+=== run: one-off dyno
+
+run.1: up for 5m: bash
+
+=== web: bundle exec thin start -p $PORT
+
+web.1: created for 30s
+
+##### with type
+
+`heroku ps web --app TestApp`
+
+=== web: bundle exec thin start -p $PORT
```

Comment:
Same for this

Reply:
Updated. All of them were in separate lines in readme file but github rendered it in a single line.